network/lib_sddf_lwip/lib_sddf_lwip.mk

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2022, UNSW
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
```

Comment:
This snippet needs a comment.  It builds `libsddf_lwip_XXX.a`  with different CFLAGS specified by SDDF_LWIP_CFLAGS_XXX etc.